eventdata/challenges/daily-log-volume-index-and-query.json

```diff
@@ -16,6 +16,31 @@
     "benchmark_type": "logs-fixed-daily-volume"
   },
   "schedule": [
+    {
+      "name": "measure-maximum-utilization",
+       "operation": {
+          "operation-type": "bulk",
+          "param-source": "elasticlogs_bulk",
+          "index": "throughput-test-elasticlogs-2999-01-01",
+          "bulk-size": {{p_bulk_size}},
+          "daily_logging_volume": "{{p_daily_logging_volume}}",
```

Comment:
yes. The reason is that we might want to lower this value for (smoke) testing the track. If this were a fixed value we'd always run for at least the time period instead of finishing quickly. As we're not after accurate values in smoke testing I think a short time period is preferable.